target/README.md

```diff
@@ -0,0 +1,40 @@
+# OpenTelemetry agent setup
+
+To enable Kloudfuse to debug customer issues telemetry from otel agent should be enabled. Otherwise only the customer will be able to view the otel telemtry metrics (sent span count, sent metrics etc)
+
+The following config needs to be enabled:
+
+
+```
+service
+  telemetry:
+    metrics:
+      address: ${MY_POD_IP}:8888
+```
+```
+receivers:
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: opentelemetry-collector
+          scrape_interval: 10s
+          static_configs:
+            - targets:
+                - ${env:MY_POD_IP}:8888
+```
+
+```
+pipelines:
+  metrics/telemetry:
+    exporters:
+      - otlphttp/customer
+      - otlphttp/kfendpoint
```

Comment:
This will go away as we want the metric to be in both local and remote control plane and this should be similar path for non-otel agents as well.